webauthn.h

```diff
@@ -93,6 +93,7 @@ extern "C" {
 //      APIs:
 //          - WebAuthNGetPlatformCredentialList
 //          - WebAuthNFreePlatformCredentialList
+//          - WebAuthNDeletePlatformCredential

```

Comment:
It seems that since the only api to get any  WebAuthnPlatformCredential is via WebAuthNGetPlatformCredentialList; what is the purpose of exposing WebAuthNDeletePlatformCredential externally? 

Does this allow the caller of WebAuthNGetPlatformCredentialList to somehow modify the returned list?  

It seems that the caller should get the list and then free the list as a whole.